tests/cisco_ios/show_tacacs/cisco_ios_show_tacacs_status_fail_count.raw

```diff
@@ -0,0 +1,32 @@
+Tacacs+ Server -  public  :
+               Server name: TACACS_1
+            Server address: 10.20.30.1
+               Server port: 49
+              Socket opens:      21752
+             Socket closes:      21752
+             Socket aborts:          0
+             Socket errors:          0
+           Socket Timeouts:          0
+   Failed Connect Attempts:          52
+        Total Packets Sent:      23450
+        Total Packets Recv:      23439
+             Server Status: Alive
+Continous Authc fail count:          2
+Continous Authz fail count:          0
+
+
+Tacacs+ Server -  public  :
+               Server name: TACACS_2
+            Server address: 10.20.30.2
+               Server port: 49
+              Socket opens:        169
+             Socket closes:        169
+             Socket aborts:          0
+             Socket errors:          0
+           Socket Timeouts:          0
+   Failed Connect Attempts:          0
+        Total Packets Sent:        174
+        Total Packets Recv:        174
+             Server Status: Alive
+Continous Authc fail count:          0
+Continous Authz fail count:          0
```

Comment:
```suggestion
Continous Authz fail count:          0

```

Reply:
I believe Marc updated the yaml with that 52 value.